### ISSUE TITLE
Membership states are not extensible.

### ISSUE THREAD

**Issue by dagrofaloyalty** [Gizra/og#214]
Detailed description is on drupal.org issue queue (https://www.drupal.org/node/2824882).
If deemed meaningful I will transfer and format it here also.

**Comment by MPParsley** [comment]
> There is no way to easily change or extend the list of the states used

Right, there isn't. But you can attach fields and have your own logic there.
